models/storyboard.js

```diff
@@ -21,14 +21,20 @@ function createNav(items) {
     const nav = [];
 
     items.forEach((item, id) => {
-        const [mediaType] = Object.keys(item);
-        // fallback for now until every type has a title field to fill in. (or something for nav)
-        const tmpTitle = item[mediaType].title ? item[mediaType].title : `${id} ${mediaType}`;
-        const title = tmpTitle.replace(/<\/?[^>]+(>|$)/g, "");
+      const [mediaType] = Object.keys(item);
+      var writeToNav = true;
+      if (item[mediaType] && item[mediaType].in_nav) {
```

Comment:
shouldn't `in_nav` here be `true`? Why do you set `writeToNav`` false`?

Reply:
Explained beneath.